wlts/collections/collection.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, name, authority_name, description, detail, datasource_id, dat
         self.scala = scala
         self.spatial_extent = spatial_extent
         self.period = period
+        self.is_public = eval(is_public)
```

Comment:
Why do you need to use `eval` expression?

Reply:
It is not necessary! I switched to the boolean type in json. Thank you! @raphaelrpl